.github/workflows/integration.yml

```diff
@@ -56,4 +56,10 @@ jobs:
       - name: Run example
         env:
           RAYON_NUM_THREADS: 2
-        run: cargo run --package ceno_zkvm --example riscv_opcodes --target ${{ matrix.target }} -- --start 10 --end 11
+        run: cargo run --release --package ceno_zkvm --example riscv_opcodes --target ${{ matrix.target }} -- --start 10 --end 11
+
+      - name: Run fibonacci
+        env:
+          RAYON_NUM_THREADS: 8
+          RUST_LOG: debug
+        run: cargo run --release --package ceno_zkvm --example fibonacci_elf --target ${{ matrix.target }} -- 
```

Comment:
@kunxian-xia Please set up your editor to end all lines (including the last line in a file) with a newline.

It's the [POSIX standard](https://stackoverflow.com/a/729795/590534) and makes GitHub and Git complain less.  Thanks!

Reply:
Thanks for your kind remind.

Reply:
Thanks.  It's only a small thing, but fixing it should be pretty quick, and will remove friction in the future.